src/SFA.DAS.EAS.Portal.UnitTests/Application/Commands/Cohort/CohortApprovalRequestedCommandHandlerTests.cs

```diff
@@ -0,0 +1,220 @@
+﻿using NUnit.Framework;
+using SFA.DAS.Commitments.Api.Client.Interfaces;
+using SFA.DAS.Commitments.Api.Types.Commitment;
+using SFA.DAS.EAS.Portal.Application.Commands.Cohort;
+using SFA.DAS.EAS.Portal.UnitTests.Builders;
+using System.Threading.Tasks;
+using FluentAssertions;
+using System.Linq;
+using System;
+using System.Collections.Generic;
+using SFA.DAS.EAS.Portal.Application.Services;
+using System.Threading;
+using SFA.DAS.EAS.Portal.Types;
+using Moq;
+using SFA.DAS.EAS.Portal.Client.Database.Models;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Application.Commands.Cohort
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CohortApprovalRequestedCommandHandlerTests
+    {
+        public class TestContext
```

Comment:
It'd be nice if these tests were converted to use `SFA.DAS.Testing` (which should be fairly easy, as these tests use a `TestContext`, which is equivalent to DAS testing's Fixture).

Although personally, I don't think having different styles of unit tests in a solution is a problem, if you prefer this style.